config/default/lazy.settings.yml

```diff
@@ -24,7 +24,7 @@ formatters:
   lazy_image: '0'
   lazy_responsive_image: '0'
 skipClass: no-lazy
-disabled_paths: "/rss.xml\r\n/entity-browser/*\r\n/entity-embed/*\r\n/media/*"
+disabled_paths: "/rss.xml\r\n/entity-browser/*\r\n/entity-embed/*\r\n/media/*\r\n/print/pdf/*"
```

Comment:
look at you. clever

Reply:
Yea I exported it to default in case anyone else uses PDFs. I don't think that will cause issue on other sites.

Reply:
For the record, I don't think entity_print is something we should enable for all sites.